src/extractor/extractor.cpp

```diff
@@ -673,25 +673,24 @@ void Extractor::WriteIntersectionClassificationData(
     util::RangeTable<> bearing_class_range_table(bearing_counts);
     file_out_stream << bearing_class_range_table;
 
-    file_out_stream << total_bearings;
+    file_out_stream.write( reinterpret_cast<const char*>(&total_bearings), sizeof(total_bearings));
     for (const auto &bearing_class : bearing_classes)
     {
         const auto &bearings = bearing_class.getAvailableBearings();
         file_out_stream.write(reinterpret_cast<const char *>(&bearings[0]),
                               sizeof(bearings[0]) * bearings.size());
     }
 
-    // FIXME
-    // This should be here, but g++4.8 does not have it...
-    // static_assert(std::is_trivially_copyable<util::guidance::EntryClass>::value,
-    //              "EntryClass Serialization requires trivial copyable entry classes");
+    if( !static_cast<bool>(file_out_stream) )
+        util::SimpleLogger().Write() << "Failed to write bearing class data.";
```

Comment:
Is this a fatal error?  If it is, we should throw an exception so everything comes crashing down and we return a bad exit code.